modules/avoindata-drupal-servicemessage/templates/avoindata_servicemessage_block.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Avoindata Servicemessage Block
+ *
+ * @ingroup themeable
+ */
+#}
+<ul class="service-messages">
+{% for message in messages %}
+  {% set alert_type = "secondary" %}
+  {% if message.severity == "critical" %}
+    {% set alert_type = "danger" %}
+  {% elseif message.severity == "warning" %}
+    {% set alert_type = "warning" %}
+  {% elseif message.severity == "info" %}
+    {% set alert_type = "info" %}
+  {% endif %}
+
+  <li class="alert alert-{{ alert_type }} alert-dismissible">
+    <span>{{ message.body|raw }}</span>
```

Comment:
It might be unwise to use raw output on this. Could `message.body|field_value` work?